development/verify-locale-strings.js

```diff
@@ -161,7 +161,7 @@ async function verifyEnglishLocale (fix = false) {
   const englishLocale = await getLocale('en')
   const javascriptFiles = await findJavascriptFiles(path.resolve(__dirname, '..', 'ui'))
 
-  const regex = /'(\w+)'/g
+  const regex = /['"](\w+)["']/g
```

Comment:
This will catch some false positives, like `"example's"` would match `example` despite the beginning quote being different from the end quote.

Something like this would work better:
```suggestion
  const regex = /'(\w+)'|"(\w+)"/g
```
However that'd create _two_ capture groups. So the `match[1]` would need to be replaced with something like
`match[1] || match[2]`